src/modules/launcher/Plugins/Microsoft.Plugin.Shell/Main.cs

```diff
@@ -418,12 +431,17 @@ public List<ContextMenuResult> LoadContextMenus(Result selectedResult)
         public void UpdateSettings(PowerLauncherPluginSettings settings)
         {
             var leaveShellOpen = false;
+            var shellOption = 0;
```

Comment:
What was the default before implementing this? I thought it was ShellExecute. We should keep the old default behavior as new default.

Reply:
Done.